autotst/conformer/systematic.py

```diff
@@ -306,43 +296,19 @@ def opt_conf(i, rmsd_cutoff):
   
```

Comment:
we should probably create a copy of the calculator
(eg. using `copy` imported from `copy` to do `copy(calc)` rather than `calc`)
so that they can each have an independent directory and aren't conflicting with each other. A single instance of a `ase.calculators.calculator.FileIOCalculator` cannot be simultaneously used by different conformers.

Reply:
hmm, I don't get what you're referring to here. Can you expand?

Reply:
All of the conformers have the same ase calculator instance. In the `opt_conf` method, we take the calculator attached to the conformer (`calculator = conformer.ase_molecule.get_calculator()`) and assign the calculator a directory to run dftb+.  However, if we run this in parallel, since all the conformers have the same calculator instance, the calc.directory will get overwritten.  We should copy the calculator so each conformer has a different instance of the calculator to prevent this from happening.

Reply:
thanks. I'll make the edits

Reply:
Has this change been implemented?